pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/mercury/ComparisonTest.java

```diff
@@ -24,9 +24,9 @@ public void testLongComparison() {
 
         ArrayList<ClinicalOrderDto> clinicalOrderDtoArrayList = new ArrayList<>();
         clinicalOrderDtoArrayList.add(new ClinicalOrderDto("FAKE_SHORT_ID", "FAKE_SAMPLE", "FAKE_ORDER", "FAKE_STATUS",
-                1L, 1L, null, "FAKE_TYPE"));
+                1L, 1L, null, "FAKE_TYPE", 1));
```

Comment:
Since we added the mercury_sequencing_id to the ClinicalOrderDto, we needed to add values to these c'tors.